src/components/MenuItem.tsx

```diff
@@ -514,6 +514,7 @@ function MenuItem(
                                         ...(Array.isArray(wrapperStyle) ? wrapperStyle : [wrapperStyle]),
                                         !focused && (isHovered || pressed) && hoverAndPressStyle,
                                         shouldGreyOutWhenDisabled && disabled && styles.buttonOpacityDisabled,
+                                        isHovered && !pressed && styles.hoveredComponentBG,
```

Comment:
[This issue](https://github.com/Expensify/App/issues/44055) was caused by the change introduced in this PR.